### PR TITLE
Update hikey960.rst

### PR DESCRIPTION
**Pull request by dk-421**
Add support for 4GB version ("ver B") of HiKey960

### CODE REVIEW

File: building/devices/hikey960.rst [hunk 1]
```diff
@@ -30,6 +32,24 @@ Build instructions
 Just follow the instructions at ":ref:`get_and_build_the_solution`". If ``make
 flash`` doesn't work, try ``make recovery``.
 
+For the 4GB RAM board version (rev B), an update to the ``CFG_DRAM_SIZE_GB`` 
+setting in `conf.mk`_ is needed. See the following patch:
```
Comment (jforissier) [inline building/devices/hikey960.rst:36]: Or simply `make PLATFORM=hikey-hikey960 CFG_DRAM_SIZE_GB=4".

Given that we now have three "flavors" which differ in UART and RAM, perhaps we should distinguish the flavors ? For instance: hikey960v1, hikey960 (that would be v2 rev A which is the target of the default build currently), hikey960v2revB.

**Review by jforissier — COMMENTED** [review]
One comment below, anyways this looks good to me.

`Acked-by: Jerome Forissier <jerome@forissier.org>`

File: building/devices/hikey960.rst [hunk 2]
```diff
@@ -30,6 +32,24 @@ Build instructions
...
+   
+   ifeq ($(PLATFORM_FLAVOR),hikey960)
+   CFG_CONSOLE_UART ?= 6
+  -CFG_DRAM_SIZE_GB ?= 3
+  +CFG_DRAM_SIZE_GB ?= 4
+   CFG_CORE_BGET_BESTFIT ?= y
+   ifeq ($(CFG_ARM32_core),y)
+   CFG_ASAN_SHADOW_OFFSET ?= 0x372E38E0
+
+
```
Comment (jbech-linaro) [inline building/devices/hikey960.rst:52]: This code block triggers an error when generating the documentation, see [Travis](https://travis-ci.org/github/OP-TEE/optee_docs/jobs/753527828).

File: building/devices/hikey960.rst [hunk 3]
```diff
@@ -30,6 +32,24 @@ Build instructions
...
+  +++ ../optee_os/core/arch/arm/plat-hikey/conf.mk	2021-01-08 13:09:32.261380826 +0100
+  @@ -48,7 +48,7 @@
+   
+   ifeq ($(PLATFORM_FLAVOR),hikey960)
+   CFG_CONSOLE_UART ?= 6
+  -CFG_DRAM_SIZE_GB ?= 3
+  +CFG_DRAM_SIZE_GB ?= 4
+   CFG_CORE_BGET_BESTFIT ?= y
+   ifeq ($(CFG_ARM32_core),y)
+   CFG_ASAN_SHADOW_OFFSET ?= 0x372E38E0
```
Comment (jbech-linaro) [inline building/devices/hikey960.rst:50]: These doesn't seem to be needed, so after a second thought, why not keep it simple and just get rid of the entire code block. You have already mentioned it at line 35,36. I believe that is sufficient.

Comment (dk-421) [reply building/devices/hikey960.rst]: Accepted. An update to line 35,36 including the DRAM size change should suffice. I will change this and at the same time remove the code block part.

File: building/devices/hikey960.rst (hunk 2 above)
Comment (dk-421) [reply building/devices/hikey960.rst]: When the code block is removed, this issue should go away .

File: building/devices/hikey960.rst (hunk 1 above)
Comment (dk-421) [reply building/devices/hikey960.rst]: I also believe that at a separate build target would be the preferred solution here.

**Comment by jbech-linaro** [comment]
@dk-421 , would you mind squash and rebase your patches (don't forget to add Jerome's Acked-by)? Once done I will merge this.

**Comment by jbech-linaro** [comment]
Due to lack of feedback and incorrect firstname, lastname I've simply reset the author to myself and added my signed-off and merged the patch. I'd be happy to at least give a Suggested-By tag, but couldn't due to missing firs and lastname.

**Pull request closed without merging**